src/component/Game.js

```diff
@@ -2,7 +2,11 @@ import React, { useState } from "react";
 import Board from "./Board";
 
 function Game() {
-  const [words] = useState("this is text".split(" "));
+  const [words] = useState(
+    "this is text that you are typing in a typing game really fast wow you are so fast at typing this is text that you are typing in a typing game really fast wow you are so fast at typing".split(
+      " "
+    )
+  );
```

Comment:
This should be in a separate commit as it doesn't directly relate to integrating the race component.

Reply:
@jgayfer this was accidentally added to the commit, it shouldn't have been included.